File: quilt/src/main/java/dev/franckyi/mcfx/quilt/MCFXModQuilt.java
package dev.franckyi.mcfx.quilt;

import dev.franckyi.mcfx.fabriclike.MCFXModFabricLike;
import org.quiltmc.loader.api.ModContainer;
import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;

public class MCFXModQuilt implements ModInitializer {
    @Override
    public void onInitialize(ModContainer mod) {
        MCFXModFabricLike.init();
    }
}

File: quilt/src/main/kotlin/dev/franckyi/mcfx/quilt/MCFXModQuilt.kt
package dev.franckyi.mcfx.quilt

import dev.franckyi.mcfx.fabriclike.MCFXModFabricLike
import org.quiltmc.loader.api.ModContainer
import org.quiltmc.qsl.base.api.entrypoint.client.ClientModInitializer

object MCFXModQuilt : ClientModInitializer {
    override fun onInitializeClient(mod: ModContainer) {
        MCFXModFabricLike.init()
    }
}